bimsb/packages/tainted.scm

```diff
@@ -196,16 +198,18 @@ deviation (SD) plots, coefficient of variation (CV) plots.")
                   "src/quantifier.pl"))
                ;; patch script using ../Rfam_for_miRDeep.fa file
                (substitute* "src/miRDeep2.pl"
-                 (("\\$\\{scripts\\}/\\.\\./Rfam_for_miRDeep\\.fa")
-                  (string-append out "/share/mirdeep2/Rfam_for_miRDeep.fa"))
-                 (("\\$scripts/\\.\\./Rfam_for_miRDeep\\.fa")
-                  (string-append out "/share/mirdeep2/Rfam_for_miRDeep.fa"))))
+                 (("\\$\\{?scripts\\}?/\\.\\.(/Rfam_for_miRDeep\\.fa)" full file)
+                  (string-append out "/share/mirdeep2" file))
```

Comment:
if you aren't using a match (like `full`) you can write `_` instead.

Reply:
I took care of this with the latest push,